src/shared/sources/in/index.js

```diff
@@ -5,13 +5,13 @@ const transform = require('../_lib/transform.js')
 
 const country = 'iso1:IN'
 
-const labelFragmentsByKey = [
-  { state: 'name of state' },
-  { deaths: 'death' },
-  { cases: 'total confirmed cases' },
-  { recovered: 'cured' },
-  { discard: 's. no.' }
-]
+const schemaKeysByHeadingFragment = {
+  'name of state': 'state',
```

Comment:
left side is a fragment of whats in the heading (will slugify and check the slug of this is uniquely inside the slug of the heading), right side is something in our schema that it maps to (or null when we want to throw out a column).